pilz_utils/include/pilz_utils/wait_for_topic.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2019 Pilz GmbH & Co. KG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef PILZ_UTILS_WAIT_FOR_TOPIC_H
+#define PILZ_UTILS_WAIT_FOR_TOPIC_H
+
+#include <string>
+
+#include <ros/ros.h>
+#include <ros/duration.h>
+
+#include <pilz_utils/wait_for_timeouts.h>
+
+namespace pilz_utils
+{
+
+/**
+ * @brief Waits until someone publishes on the specified topic.
+ */
+template<class T>
+static void waitForTopic(const std::string topic_name,
```

Comment:
This function should really have a better name. See the diff in comment and function name?

Reply:
Done: https://github.com/PilzDE/pilz_robots/pull/278